PodioKit/Common/Models/PKTCalendarEvent.m

```diff
@@ -69,4 +72,62 @@ + (PKTAsyncTask *)fetchAllFromDate:(NSDate *)fromDate toDate:(NSDate *)toDate pr
   return task;
 }
 
++ (PKTAsyncTask *)fetchEventsFromDate:(NSDate *)fromDate toDate:(NSDate *)toDate {
+  return [self fetchEventsFromDate:fromDate toDate:toDate priority:0];
+}
+
++ (PKTAsyncTask *)fetchEventsFromDate:(NSDate *)fromDate toDate:(NSDate *)toDate priority:(NSUInteger)priority {
+  PKTRequest *request = [PKTCalendarAPI requestForGlobalCalendarWithFromDate:fromDate toDate:toDate priority:priority includeTasks:NO];
+  PKTAsyncTask *requestTask = [[PKTClient currentClient] performRequest:request];
+  
+  Class objectClass = [self class];
+  
+  PKTAsyncTask *task = [requestTask map:^id(PKTResponse *response) {
```

Comment:
These methods all seem pretty similar too. Is generating the request the only thing unique?